main.nf

```diff
@@ -124,7 +125,7 @@ Channel
     .groupTuple(sort: true)
     .set { read_files }
 
-read_files.into { fastQC_input; trimgalore_input; name_for_bowtie_miRBase_mature; name_for_bowtie_miRBase_hairpin; name_for_bowtie2; name_for_samtools }
+read_files.into { fastQC_input; trimgalore_input; name_for_fastqc; name_for_trimgalore; name_for_bowtie_miRBase_mature; name_for_bowtie_miRBase_hairpin; name_for_bowtie2; name_for_samtools }
```

Comment:
See below comment about `tag "$name"` - I think we may be able to get rid of all of the `name_for_....` channels here.